apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -98,6 +100,10 @@ export class DotEditContentHtmlService {
         }
     }
 
+    setCurrentPage(page: DotPage) {
```

Comment:
doc?

Reply:
You need to test this one.

Reply:
done https://github.com/dotCMS/core-web/pull/1803/commits/7fba5d5be372b9a2235170b8758f0d224ff56026#diff-52eac63ed3005a5954cc32e0693785021e935a08dd1cfc146b43eb98f76576d8R103

done https://github.com/dotCMS/core-web/pull/1803/commits/7fba5d5be372b9a2235170b8758f0d224ff56026#diff-2263dd4c5322cf90f7ff085645ababce8caf2e24d868711bcf585bc6aeddd7c4R370

removing d: https://github.com/dotCMS/core-web/pull/1803/commits/c4494f526443b1c508c2e9fc9b91b75a5935ea81#diff-2263dd4c5322cf90f7ff085645ababce8caf2e24d868711bcf585bc6aeddd7c4R370